.github/workflows/build.yml

```diff
@@ -105,6 +105,49 @@ jobs:
         with:
           name: lovr.apk
           path: build/lovr.apk
+  magicleap:
+    name: Magicleap
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Update Packages
+        run: sudo apt update
+      - name: Install Packages
+        run: sudo apt install -y glslang-tools
+      - name: Install Android 29
+        run: ${ANDROID_HOME}/cmdline-tools/latest/bin/sdkmanager --install "platforms;android-29"
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: 'recursive'
+      - name: Init
+        run: >
+          mkdir build &&
+          cd build &&
+          keytool
+          -genkey
+          -dname 'cn=Unknown, ou=Unknown, o=Unknown, l=Unknown, st=Unknown, c=Unknown'
+          -keystore key.keystore
+          -keypass hunter2
+          -storepass hunter2
+          -alias key
+          -keyalg RSA
+          -keysize 2048
+          -validity 10000 &&
+          cmake ..
+          -D CMAKE_TOOLCHAIN_FILE=$ANDROID_NDK/build/cmake/android.toolchain.cmake
+          -D ANDROID_SDK=$ANDROID_HOME
+          -D ANDROID_ABI=x86_64
```

Comment:
Since this is the same as the main Android job except for the ABI, we could probably use a [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) for it.  Basically there would be a matrix section at the top:

```yaml
android:
  name: Android
  strategy:
    matrix:
      abi: [arm64-v8a, x86_64]
```

And then add in `${{ matrix.abi }}` when setting `ANDROID_ABI` and the APK name.  I can do this as a future improvement if you want though.

Reply:
that would be great, I am not super familiar with github workflows